src/component/common/Table/TableSearch/TableSearch.tsx

```diff
@@ -16,14 +18,37 @@ interface ITableSearchProps {
 export const TableSearch: FC<ITableSearchProps> = ({
     initialValue,
     onChange = () => {},
-    placeholder = 'Search',
+    placeholder: customPlaceholder,
 }) => {
     const [searchInputState, setSearchInputState] = useState(initialValue);
     const [isSearchExpanded, setIsSearchExpanded] = useState(
         Boolean(initialValue)
     );
     const [isAnimating, setIsAnimating] = useState(false);
+    const [isTooltipOpen, setIsTooltipOpen] = useState(false);
     const debouncedOnSearch = useAsyncDebounce(onChange, 200);
+    useKeyboardShortcut({ modifiers: ['ctrl'], key: 'f' }, () => {
+        setIsTooltipOpen(true);
+    });
+    useKeyboardShortcut(
+        { modifiers: ['ctrl'], key: 'k', preventDefault: true },
+        () => {
+            setIsSearchExpanded(true);
+        }
+    );
+    useKeyboardShortcut({ key: 'Escape' }, () => {
+        if (isSearchExpanded) {
+            setIsSearchExpanded(false);
+        }
+    });
+    const isAppleDevice = useIsAppleDevice();
+    const placeholder =
+        customPlaceholder ??
+        `Search${
+            isAppleDevice !== undefined
+                ? ` (${isAppleDevice ? '⌘' : 'Ctrl'}+K)`
```

Comment:
I believe this is the wrong character for Ctrl in Mac. It should be something like this instead:
```suggestion
                ? ` (${isAppleDevice ? '⌃' : 'Ctrl'}+K)`
```

Reply:
But on mac, "Command+..." shortcuts are standard in system, and there is a check in "useKeyboardShortcut" remapping it

Reply:
I refactored the hook - not it returns the hotkey string :)